lib/skate/settings/db/user.ex

```diff
@@ -19,6 +20,10 @@ defmodule Skate.Settings.Db.User do
     has_many(:notification_users, DbNotificationUser)
     many_to_many(:notifications, DbNotification, join_through: DbNotificationUser)
     has_many(:route_tabs, DbRouteTab, on_replace: :delete_if_exists)
+
+    # :test_groups goes through :test_group_users to find the :test_group's the user belongs to
```

Comment:
I don't personally feel like this needs a comment, given that you can see it's used in the `has_many` declaration for `:test_groups`, though I could be convinced otherwise if it still feels highly unintuitive to others.

Reply:
I found myself very confused when first reading over this, but this feels like a special case to elixir beginners.

It's personally confusing how something inside a function call can become a referencable member, but we don't document this in all the other similar places, so I don't believe it needs to stay.